docs/governance/actions.md

```diff
@@ -1,12 +1,12 @@
-# ODS Board Actions
+# TODS Board Actions
 
-Record of actions taken by the ODS Board.
+Record of actions taken by the TODS Board.
 
 *Note that routine actions such as approval of meeting minutes are not listed here*
 
 | **Date** | **Action** | **Record (Yes - No - Abstain)** |
 | -------- | ---------- | ---------- |
-| 6/26/24 | TODS Program Manager granted authority to make non-substantive changes to TODS governance to fix typos and change spec branding from ODS to TODS. | (5 - 0 - 0) |
```

Comment:
I think the last part of this sentence needs to stay saying "from ODS to TODS."

Reply:
Fixed @jfabi